packages/inter-protocol/test/auction/test-auctionContract.js

```diff
@@ -185,13 +185,23 @@ const makeAuctionDriver = async (t, customTerms, params = defaultParams) => {
         give: { Collateral: collateralAmount },
       }),
       harden({ Collateral: collateralPayment }),
+      limit,
     );
     await eventLoopIteration();
 
     return seat;
   };
 
-  const setupCollateralAuction = async (issuerKit, collateralAmount) => {
+  /**
+   * @param {Pick<IssuerKit<'nat'>, 'brand' | 'issuer' | 'mint'>} issuerKit
+   * @param {Amount<'nat'>} collateralAmount
+   * @param {undefined | { goal: Amount<'nat'> }} limit
```

Comment:
ditto re: name and our style is to express optionality with brackets (per JSDoc)
```suggestion
   * @param {{ goal: Amount<'nat'> }} [offerArgs]
```